src/main/res/layout/log_activity.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <EditText
```

Comment:
Any reason you're using an EditText? This shows autocomplete errors and lets the user change stuff. TextView should be fine. Also, text is weirdly centered, you should probably use `gravity=top`.